app/models/setting/rh_cloud.rb

```diff
@@ -7,6 +7,7 @@ def self.default_settings
       set('allow_auto_insights_sync', N_('Allow recommendations synchronization from Red Hat cloud'), false),
       set('obfuscate_inventory_hostnames', N_('Obfuscate host names sent to Red Hat cloud'), false),
       set('rh_cloud_token', N_('Authentication token to Red Hat cloud services. Used to authenticate requests to cloud APIs'), 'DEFAULT', N_('Red Hat Cloud token'), nil, encrypted: true),
+      set('include_parameter_tags', N_('Should import include parameter tags from Foreman?'), false),
```

Comment:
Adding this as a boolean setting, not that the [original](https://github.com/redhataccess/foreman-plugin/blob/master/redhat-access/app/controllers/redhat_access/api/machine_telemetry_api_controller.rb#L175) could be `true`, `false`, or array of values to filter by, which is not compatible with our setting types at all. Do we need that granularity? Not sure how many people were using it since there was not even an example for that.